plugins/baseball-scoreboard/logo_manager.py

```
@@ -183,6 +185,82 @@ def load_milb_logo(self, team_abbr: str, logo_dir: Path) -> Optional[Image.Image
+        safe_id = self._safe_filename(player_id)
+        safe_league = self._safe_filename(league) or "unknown"
+        disk_path = None
+        if safe_id:
+            disk_path = self._HEADSHOT_DIR / safe_league / f"{safe_id}.png"
+            if disk_path.exists():
+                try:
+                    with Image.open(disk_path) as src:
+                        img = self._crop_square(src.convert("RGBA"), max_size)
+                    self._logo_cache[cache_key] = img
+                    return img
+                except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)